### PR TITLE
Is Factorio an annoying game?

### PR DESCRIPTION
**Pull request by MrTrupen**
### Author info
Name: trupen
Suggested release date:

### CODE REVIEW

**Comment by ickputzdirwech** [comment]
I am not that familiar with GitHub, so I am not sure if this is the right place to put this. I just read your piece and I quite like it. I have one remark about the power pole spaghetti though: Do you know that the devs are addressing this issue for 1.1.0? See [here](https://www.reddit.com/r/factorio/comments/ijf3qw/why_arent_copper_wires_treated_the_same_a_signal/g3gbobb/) and [here](https://www.reddit.com/r/factorio/comments/ijf3qw/why_arent_copper_wires_treated_the_same_a_signal/g3h58nz/)

File: submissions/no pain no gain/no pain no gain.md [hunk 1]
```diff
@@ -0,0 +1,78 @@
...
+         "Trees and removing them" : 168
+         "The entire fluid system" : 225
+         "Power poles spaghetti connecting" : 397
+```
+
+Thanks to every 1.6k people who took part in the survey and expressed their opinion. Results shows that the most annoying things for players is "Power poles spaghetti connecting". I don't think anyone was surprised that power poles will be one of the most annoying thing, because everyone try to keep their factory in perfect by using good ratios, proper trains names or by building a "main bus" design.
+
+![](https://i.imgur.com/Ey0mvSp.png)
+
+Blueprints however don't save connected cables and no matter how pretty they look when you creating a blueprint, this one little detail will not be saved. It's getting more noticeable, when you're entering megabase stadium and most of your buildings are made out of blueprints.
```
Comment (Conor12345) [inline submissions/no pain no gain/no pain no gain.md]: Need to add a note here that this is going to be fixed in 1.1 *i think*, need to verify though.

File: submissions/no pain no gain/no pain no gain.md [hunk 2]
```diff
@@ -0,0 +1,78 @@
...
+
+![](https://i.imgur.com/x8jG8go.png)
+
+### Painful things that got removed
+
+Every game developer tries to make their game the best on the market. Factorio devs put a lot of effort to make their game very intuitive and fun to play without taking advanced mechanics from the game. I think many people would agree that factorio is "easy to play, hard to master" type of game, because you can start playing it without any knowledge and know what you are doing and what you need to do next, but it wasn't always like that.
+
+Factorio had a lot of annoying mechanics, that every player hated. I don't know if many of you remember killing biter bases to get some [Alien artifact](https://wiki.factorio.com/Alien_artifact), because that feature was deleted in 0.15 (24.04.2017). For those who don't remember, alien artifacts was used to craft advance materials like modules tier 3 or power armors and also you could craft [purple science](https://wiki.factorio.com/Alien_science_pack), that was the highest tier of science packs in Factorio version 0.15. Constant expansion for more purple orbs wasn't in "Factorio style" where idea is to automate everything, so they got removed and very well.
```
Comment (Conor12345) [inline submissions/no pain no gain/no pain no gain.md]: Possibly reference the nauvis archives about purple goop from a previous issue here?

File: submissions/no pain no gain/no pain no gain.md [hunk 3]
```diff
@@ -0,0 +1,78 @@
...
+[Steel axe](https://wiki.factorio.com/Steel_axe) on the other hand was with us throughout the whole game, crafted from time to time in a bulk amount.
+
+![](https://i.imgur.com/jtQxq0U.gif)
+
+There is more things like: old toolbar, old blueprints management, not upgrade planner, old oil processing, fluids system, no auto replacing belts or homing biters shots. All of this and many more was removed/upgraded during factorio early access. Many games in alpha/beta stage gets a lot of new buildings, mechanics or enemies, but factorio chose difference path and instead of adding more they improved what is already there. People who feel lack of content in factorio can always finds something in huge [library of mods](https://mods.factorio.com/), because stable core makes modding easier.
+
+If you want to read more about removed mechanics there is old fff [Cleanup of mechanics](https://factorio.com/blog/post/fff-266), that will explain in detail decisions behind: pickaxes, furnace heating, burner efficiency and ore hardness.
```
Comment (Conor12345) [inline submissions/no pain no gain/no pain no gain.md]: These outdated mechanics are also going to be talked about here: https://github.com/AlternativeFFFF/Alt-F4/pull/227.
Could this section be replaced by something else that is more related to the start of the submission?

File: submissions/no pain no gain/no pain no gain.md [hunk 4]
```diff
@@ -0,0 +1,78 @@
...
+
+
+### All your factorio pain in one video...
+You might already saw this video, because it was quite popular on reddit and now is going viral on YouTube, but if not then this video collected (almost) all painful things from factorio like <s>xterminator</s> being killed by a train or looking at your stuck spidertrone, into fast paced compilation.
```
Comment (Conor12345) [inline submissions/no pain no gain/no pain no gain.md]: Due to the delay in the release, this section needs rewording to past tense. Maybe mention the part 2 video instead?

File: submissions/no pain no gain/no pain no gain.md [hunk 5]
```diff
@@ -0,0 +1,78 @@
...
+![](https://i.imgur.com/q9qlUXa.gif)
+
+With the help of the above-mentioned technics, I've build very interesting machines that are useless in every aspect and their only purpose was to deal maximum amount of pain. There is a small list with explanation.
+I wanted to show cumulative "Train out of fuel" alert and to do this, I simply stopped time in editor and make 10 pointless rails circles. In every circle, I've placed one locomotive and setup schedule for infinite repetitions. As time was stopped, there was no problem to put one more wood to each subsequent locomotive. When everything was ready, I exited editor mode to start time and recorded appearing pop-ups.
+
+![](https://i.imgur.com/i40EZ8C.gif)
+
+Second machine was used to make yt thumbnail and it's just four locomotives connected to one circuit condition and set up to travel straight no matter what. You can also see simple self-timer made out of belts to have time for taking best position.
+
+![](https://i.imgur.com/E2Gatvk.gif)
```
Comment (Conor12345) [inline submissions/no pain no gain/no pain no gain.md]: Each image will need to be moved to media server by editors and alt-text added for accesibility

File: submissions/no pain no gain/no pain no gain.md [hunk 6]
```diff
@@ -0,0 +1,78 @@
...
+
+### No pain, No gain
+Factorio has come a long way (8.5 years) to be right where it is. Mistakes were made, but devs didn't pretend that everything is fine and they work hard to make factorio a wonderful games for casual players and veterans. From this submission you can think than I'm tormenting this game and looking for everything I can attach to, but truth is I respect work that Wube software [team](https://factorio.com/game/about) put into this masterpiece.
+
+In my next submission I'll try to explain why factorio is such a perfect game instead of saying what is wrong I'll say what is good. I'm not a member of Wubue software, but my knowledge and experience is enough to revers-engineer the best parts of the game. It will only be the beginning of the iceberg, because 8.5 years of development can't be shrink to one little post.
```
Comment (Conor12345) [inline submissions/no pain no gain/no pain no gain.md]: Not sure this part is neccesary, when the next part comes out, we can refer back to this one instead i think.

File: submissions/no pain no gain/no pain no gain.md (hunk 3 above)
Comment (ClaudeMetz) [reply submissions/no pain no gain/no pain no gain.md]: I'll keep this in as it isn't duplicated in #227. I think it somewhat fits with this.

**Comment by ClaudeMetz** [comment]
Okay I took a look at this, I really like the content, I just had to make quite a few adjustements to the phrasing and some to the structure. I also took out the part about creating the video because I don't think it really fits with the rest of the submission. I'm totally open to including it as a separate piece in the future though, we'll see.

Thanks for the submission Trupen, and sorry that it took this long for me to get to it.

**Pull request merged**